nservicebus/msmq/subscription-persistence_config_core_[5,6].partial.md

```diff
@@ -0,0 +1,11 @@
+WARNING: Storing subscriptions in MSMQ is not suitable when endpoints are scaled-out, because the subscription queue cannot be shared among multiple endpoints.
+
+To configure MSMQ as the subscription persistence:
+
+snippet:ConfiguringMsmqPersistence
+
+By default NServiceBus uses a queue called `NServiceBus.Subscriptions`. If not specified otherwise, all endpoints will share that queue to store subscriptions. In order to specify a different queue, add the following `configSections` and subsequent config entry:
```

Comment:
> By default NServiceBus uses a queue called `NServiceBus.Subscriptions`. If not specified otherwise, all endpoints will share that queue to store subscriptions. In order to specify a different queue, add the following `configSections` and subsequent config entry:

Can we combine this and the warning to make it read better? For example, how about:

When using MSMQ Subscription persistence, the default queue to store the subscription messages is `NServiceBus.Subscriptions`. It is important to override this queue name for each endpoint to avoid sharing the same subscription queue for different endpoints running on the same machine. To specify a different subscriptions queue, add the following configuration as shown below: